packages/logging/src/transports/index.ts

```diff
@@ -0,0 +1,12 @@
+import { LogEvent } from '..';
+
+export interface Transport {
+  log: (logs: LogEvent[]) => Promise<void> | void;
+  flush: () => Promise<void> | void;
+}
+
+export * from './axiom-js';
```

Comment:
I don't like how the index file is import/export other transports, and the other transports are importing the Transport interface, cycle dependencies are always weird. I would prefer if the Transport interface lived in its own file in the src folder.